base/simplebook/ansible/inv/staging/hosts

```diff
@@ -0,0 +1,2 @@
+[simplebook]
```

Comment:
Similar to the dev environment, this environment is actually defunct.  I'm actually going to go through and remove them from mwlib and torquedata this morning!

Reply:
Just removed it ty